commit 3f0c0a2fb522ee42509e752be7a5af2d6206f293 (refs/changes/53/2953/4)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2017-11-16T16:11:13-08:00 (1 year, 11 months ago)
    
    joyent/manta-thoth#154 Need license info in package.json
    Reviewed by: Bryan Cantrill <bryan@joyent.com>
    Approved by: Bryan Cantrill <bryan@joyent.com>

diff --git a/package.json b/package.json
index 3fcb1fc..54eb376 100644
--- a/package.json
+++ b/package.json
@@ -2,6 +2,7 @@
 	"name": "manta-thoth",
 	"description": "Manta-based system for core and crash dump analysis",
 	"version": "1.0.0",
+    "license": "MIT",
 	"author": "Joyent (joyent.com)",
 	"dependencies": {
 		"manta": "^4.0.0",
